rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -37,6 +37,12 @@ extern void ppu_register_function_at(u32 addr, u32 size, ppu_intrp_func_t ptr);
 
 extern void sys_initialize_tls(ppu_thread&, u64, u32, u32, u32);
 
+std::unordered_map<std::string, ppu_static_module*>& ppu_module_manager::get()
+{
+	static std::unordered_map<std::string, ppu_static_module*> s_module_map;
+	return s_module_map;
+}
```

Comment:
There should at least be a comment in the code that explains this. Otherwise the next person might just make it a static member again.

Reply:
OK, done.